1-setup-windows-msys2.sh

```diff
@@ -16,6 +16,7 @@ SCRIPT_DIR=`dirname "$0"`
 source ${SCRIPT_DIR}/autobuild/msys2/set_env.sh
 
 echo "Selected ARCH: ${ARCH}"
+echo "MINGW_PACKAGE_PREFIX: ${MINGW_PACKAGE_PREFIX}"
```

Comment:
Not sure what this is for, but it shows blank in the logs [here](https://ci.appveyor.com/project/Synfig/synfig/builds/34428770/job/68euae6snrdc960x?fullLog=true#L11) (Even for 32 bit one [here](https://ci.appveyor.com/project/Synfig/synfig/builds/34428770/job/sk63ob2yoqha3m31?fullLog=true#L11)).

Reply:
Fixed